tfe/data_source_ssh_key.go

```diff
@@ -0,0 +1,63 @@
+package tfe
+
+import (
+	"fmt"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceTFESSHKey() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFESSHKeyRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
No need for `ForceNew` on data sources.

Reply:
Removed `ForceNew` from all data sources.